content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -21,33 +21,60 @@ further_reading:
 
 - The .NET Tracer library for Datadog is open source. For more information, see the [.NET Tracer repository][1].
 
+## Runtime support policy for .NET Core APM
+
+Datadog APM for .NET Core is built upon dependencies defined in specific versions of the host operating system, .NET Core runtime, certain .NET Core libraries, and the Datadog Agent/API. When these versions are no longer supported by their maintainers, Datadog APM for .NET Core limits its support for these as well.
+
+### Levels of support
+
+delner marked this conversation as resolved.
+
+| **Level**                                              | **Support provided**                                                                                                                                                          |
+|--------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      |  No implementation. [Contact our customer support team for special requests.][2]                                                             |
+| <span id="support-beta">Beta</span>                    |  Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
+| <span id="support-ga">General Availability (GA)</span> |  Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
+| <span id="support-maintenance">Maintenance</span>      |  Full implementation of existing features. Does not receive new features. Support for bug & security fixes only.                                                              |
+| <span id="support-legacy">Legacy</span>                |  Legacy implementation. May have limited function, but no maintenance provided. [Contact our customer support team for special requests.][2] |
+| <span id="support-eol">End-of-life (EOL)</span>        |  No support.                                                                                                                                                                  |
```

Comment:
I don't think in .NET we need all of these. For example, 1.x is currently in Maintenance and then will go into either Legacy or EOL, but we don't need both of those.

Reply:
(the RFC allows for this flexibility, not all language tracers need all levels)

Reply:
Yes I removed legacy at first, and then I thought ok if we need it one day I won't need to add it. 
But Ok, i'll remove it